src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/Binders/PlaceholderLocalBinder.cs

```diff
@@ -116,17 +116,18 @@ private PlaceholderLocalSymbol LookupPlaceholder(string name)
 
             switch (kind)
             {
-                case PseudoVariableKind.Exception:
-                case PseudoVariableKind.StowedException:
-                    return new ExceptionLocalSymbol(_containingMethod, id, type);
-                case PseudoVariableKind.ReturnValue:
-                    return new ReturnValueLocalSymbol(_containingMethod, id, type, index);
-                case PseudoVariableKind.ObjectId:
-                    return new ObjectIdLocalSymbol(_containingMethod, type, id, isWritable: false);
-                case PseudoVariableKind.DeclaredLocal:
-                    return new ObjectIdLocalSymbol(_containingMethod, type, id, isWritable: true);
-                default:
-                    throw ExceptionUtilities.UnexpectedValue(kind);
+            case PseudoVariableKind.Exception:
```

Comment:
There was a bug for a while where the default for an option was wrong, and when we fixed it many people had the wrong value persisted already.  Go to Tools\Options\Text Editor\C#\Formatting\Indentation and make sure both "Indent case labels" and "Indent case contents" are checked.